ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPartyManagementService.scala

```diff
@@ -393,10 +406,10 @@ private[apiserver] final class ApiPartyManagementService private (
 private[apiserver] object ApiPartyManagementService {
 
   private def toProtoPartyDetails(
-      partyDetails: domain.PartyDetails,
+      partyDetails: IndexerPartyDetails,
       metadataO: Option[ObjectMeta],
-  ): PartyDetails =
-    PartyDetails(
+  ): proto_admin.party_management_service.PartyDetails =
```

Comment:
That's a bit of a mouthful could we instead introduce aliases
```
ProtoPartyDetails
ProtoObjectMeta
```
?

Reply:
done